src/components/Verses/BookmarkPill.tsx

```diff
@@ -0,0 +1,54 @@
+import { useContext } from 'react';
+
+import useTranslation from 'next-translate/useTranslation';
+
+import styles from './BookmarkPill.module.scss';
+
+import DataContext from '@/contexts/DataContext';
+import Link from '@/dls/Link/Link';
+import CloseIcon from '@/icons/close.svg';
+import { getChapterData } from '@/utils/chapter';
+import { logButtonClick } from '@/utils/eventLogger';
+import { toLocalizedVerseKey } from '@/utils/locale';
+import { getVerseNavigationUrlByVerseKey } from '@/utils/navigation';
+import { getChapterNumberFromKey } from '@/utils/verse';
+
+interface Props {
+  verseKey: string;
+  onDeleted: (verseKey: string) => void;
+}
+
+export default function BookmarkPill({ verseKey, onDeleted }: Props) {
```

Comment:
Nit-pick: We declare new components like this 

`const BookmarkPill: React.FC<Props> = ({})=> { ...`

then 

`export default BookmarkPill` at the bottom.